src/Server/Repository/RefreshTokenRepositoryInterface.php

```diff
@@ -18,9 +18,16 @@
 
 namespace ZfrOAuth2\Server\Repository;
 
+use ZfrOAuth2\Server\Model\RefreshToken;
+
 /**
  * Interface RefreshTokenRepositoryInterface
  */
 interface RefreshTokenRepositoryInterface extends TokenRepositoryInterface
```

Comment:
I'm not sure it makes sense now to have a TokenRepositoryInterface considering that we have specialized interface. Then you would be able to typehint other methods (deleteToken, tokenExists...) using the concrete model.